validator/connection/src/lib.rs

```diff
@@ -57,6 +58,7 @@ impl ValidatorConnection {
             uri: uri.clone(),
             validator_api_client,
             blockchain_api_client,
+            logger,
```

Comment:
Would it make sense to add context to this logger so that it says that it's for the validator rather than adding "validator" to all the logged warnings/errors?

Reply:
In my experience the context is not great for immediately communicating information to users, since its usually the message that gets looked at and not the custom fields.